File: Assignment No_39/Question_2.java
 /* 2. Write java program which accept N numbers from user and accept one another number as NO , return index of first occurrence of that NO.

Input : N : 6
        NO: 66
        Elements :85 66 3 66 93 88
Output : 1

Input : N : 6
        NO: 12
        Elements :85 11 3 15 11 111
Output : -1
*/
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

 
 import java.util.*;

class Ass39_2
{
    public static int Check_Index(int Brr[], int iNo)
    {
        int i = 0;
        int iResult =0;

        for(i = 0; i < Brr.length; i++)
        {
            if(Brr[i] == iNo)
            {
                iResult = i;
                break;
            }
        }

        return iResult;
    }

    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in); 
        int i = 0, iSize = 0, iValue = 0;
        int iRet = 0;

        System.out.println("Enter the number of elements : ");
        iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter the values : ");

        for(i = 0; i < iSize; i++)
        {
            Arr[i] = sobj.nextInt();
        }

        System.out.println("Elements of the array are : ");

        for(i = 0; i < iSize; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();

        System.out.println("Enter the element that you want to search : ");
        iValue = sobj.nextInt();

        iRet = Check_Index(Arr,iValue);

        System.out.println(+iRet);
    }
}

File: Assignment No_37/Question_2.c
/* 2. Write Java program which accept N numbers from user and display all such elements which are divisible by 5.

Input : N : 6
       Elements :85 66 3 80 93 88
Output : 85 80

*/
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class ArrayX
{
    public int Arr[];

    public ArrayX(int no)
    {
        Arr = new int[no];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the elements : ");
        int i = 0;

        for(i = 0; i < Arr.length; i++)
        {
            Arr[i] = sobj.nextInt();
        }
    }

    public void Display()
    {
        System.out.println("Elements of the array are : ");
        int i = 0;

        for(i = 0 ; i < Arr.length; i++)
        {
            System.out.println(Arr[i]);
        }
    }

    public void Check_Divisible_By_Five()
    {
        int  i = 0;
        int iAns = 0;

        for(i = 0 ; i < Arr.length; i++)
        {
           if(Arr[i] % 5 == 0)
           {
             System.out.println(+Arr[i]);
           }
            
        }

    }
}

class Ass37_2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements you want to store : ");
        int iNo = sobj.nextInt();

        ArrayX aobj = new ArrayX(iNo);

        System.out.println("Array size is : "+aobj.Arr.length);

        aobj.Accept();

        aobj.Display();

        aobj.Check_Divisible_By_Five();

    }
}

















